src/components/sidebar/linkView/LinkView.tsx

```diff
@@ -121,13 +121,11 @@ class LinkView extends React.Component<ILinkViewProps, ILinkViewState> {
                         />
```

Comment:
Why sometimes use `<InputContainer>` and other times `<div>` and `<input>`?

Reply:
Not part of this PR. This is only for fixing dropdown. See card #283

Reply:
👍